ui/src/app/components/apps-panel/expanded-app.tsx

```diff
@@ -27,7 +27,7 @@ import colors from 'app/styles/colors';
 import { reactStyles } from 'app/utils';
 import { setSidebarActiveIconStore } from 'app/utils/navigation';
 import {
-  isActionable,
+  canDeleteRuntime,
```

Comment:
`isActionable` should be renamed, or its comment should be reworded, since I'd consider "deletion" an action.

Reply:
good call.  I have an idea for that in a small future PR.

Reply:
#8100